### ISSUE TITLE
Arena doesn't support multi kubeconfig file

### ISSUE THREAD

**Issue by asdfsx** [kubeflow/arena#51]
My KUBECONFIG config as follows
```
minikube=~/.kube/config
kubenetes=~/.kube/localconfig
hongkong=~/.kube/hongkongconfig
export KUBECONFIG=$KUBECONFIG:$minikube:$kubenetes:$hongkong\
```
but when I execute`arena list`, I got the following error
```
WARN[0000] Illegal kubeconfig file: :~/.kube/config:~/.kube/localconfig:~/.kube/hongkongconfig
FATA[0000] stat :~/.kube/config:~/.kube/localconfig:~/.kube/hongkongconfig: no such file or directory
```

**Comment by cheyang** [comment]
`arena list` only supports single cluster.

**Comment by asdfsx** [comment]
Is there any plan to support multi kubeconfig file or multi context?

**Comment by wsxiaozhang** [comment]
@asdfsx yes, we have plan to wrapper a arena context which is not only for clusters env but also security . pls stay tuned.

**Comment by chacha923** [comment]
hello，when this feature will supported？i can't also create multi arena client by Golang SDK, because arena use once.Do() method to init @wsxiaozhang

**Comment by chacha923** [comment]
and i think use environment a bad measure，because i may use lock when concurrency request

**Comment by zwbdzb** [comment]
is the feature will be update?     

The golang  sdk do not support multi  cluster .
> The golang sdk  still  use  once.Do, so the arenaClient is just a singleton pattern.

Please document the current situation, Currently golang  sdk  is just a demo, I‘m depressed. @wsxiaozhang
